File: sentimental-mario-less/mario.py
# TODO

from cs50 import get_int

while True:

    # input height
    n = get_int("Height: ")
    # upto 9 height
    if n > 0 and n < 9:
        break
for i in range(0, n, 1):
    for j in range(0, n, 1):
        if (i + j < n - 1):
            print(" ", end="")
        else:
            print("#", end="")
    print()

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n, i, j, k;
    do
    {
        //take user input
        n = get_int("Enter height here :"); // n is height
    }
    while (n < 1 || n > 8);

    for (i = 0; i < n; i++) // i is row
    {
        for (j = 0; j < n - i - 1; j++) // j is space
        {
            printf(" ");
        }
        for (k = 0; k <= i ; k++) // k is column
        {
            printf("#");
        }
        printf("  "); // for space between two pyramide
        for (k = 0; k <= i; k++)
        {
            printf("#"); //printing right side
        }
        printf("\n");
    }
}